Error: Cannot find module 'tmp'

```
$ npm install -g @eclipse-che/theia-generator
$ npm install -g che:theia init -d
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'tmp'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/monaka/.npm-global/lib/node_modules/@eclipse-che/theia-generator/dist/init-sources.js:27:13)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

--- a/package.json
+++ b/package.json
@@ -17,7 +17,8 @@
     "yargs": "12.0.5",
     "webpack": "^4.0.0",
     "html-webpack-plugin": "^3.2.0",
-    "axios": "0.18.0"
+    "axios": "0.18.0",
+    "tmp": "^0.0.33"
   },
   "devDependencies": {
     "@types/fs-extra": "5.0.4",
@@ -33,7 +34,6 @@
     "json2yaml": "^1.1.0",
     "nexe": "^2.0.0-rc.34",
     "rimraf": "2.6.2",
-    "tmp": "^0.0.33",
     "ts-jest": "23.10.5",
     "tslint": "5.11.0",
     "typescript": "3.1.6",

